docs/src/pages/customization/css-in-js.md

```diff
@@ -91,10 +92,11 @@ const styleSheet = createStyleSheet('MyLink', (theme) => ({
 ### `withStyles(styleSheet, [options]) => Higher-order Component`
 
 Link a style sheet with a component.
-It does not modify the component passed to it; instead, it returns a new, with a `classes` property.
+It does not modify the component passed to it; instead, it returns a new component, with a `classes` property.
 This `classes` object contains the name of the class names injected in the DOM.
-Some implementation details that might be interested in being aware of:
- - It's forwarding not react static properties so this HOC is more "transparent".
+
+Some implementation details that might be interesting to being aware of:
+ - It's not forwarding react static properties so this HOC is more "transparent".
```

Comment:
I think that it should be the opposite as we are using https://github.com/mridgway/hoist-non-react-statics
`It's forwarding non react static properties so this HOC is more "transparent".`